storybook/cypress/e2e/autosuggest.cy.ts

```diff
@@ -141,15 +141,17 @@ describe("Autosuggest", () => {
     cy.get("dso-autosuggest.hydrated").find("div[role='option']").eq(1).should("contain", "adres in Rotterdam");
   });
 
-  it("suggestOnFocus should show suggestions on focus by keyboard", { browser: "!firefox" }, () => {
+  it("suggestOnFocus should not show suggestions on focus by keyboard", { browser: "!firefox" }, () => {
```

Comment:
Dit waren eigenlijk twee testen. Eentje faalt er flaky. Door er twee losse tests van te maken hebben we meer debugmateriaal bij de volgende flaky